core/.changelog.d/4351.changed.3

```diff
@@ -0,0 +1 @@
+Forbidden per-node paths in multisig.
```

Comment:
Also "Forbid ..." or "Forbade ...", preferably the former. Also this concerns only change outputs and GetAddress, so I'd be specific about that. Maybe "Forbid different sub-paths in multisig change-outputs and receive addresses."

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4351/commits/c5200261fbe0fa28d0c2f826f472e1336cba674c.